README.adoc

```diff
@@ -27,15 +40,15 @@ If you find yourself building GHC, STOP and fix the cache.
 
 == How to use the project
 
-This section contains brief information about how to use this project. For development
-work see <<how-to-develop>> for more information.
+This section contains brief information about how to use this project.
+For development work see <<how-to-develop>> for more information.
 
 [[prerequisites]]
 === Prerequisites
 
-The Haskell libraries in the Plutus Platform can be built in a number of ways. The prerequisites depend
-on how you want to build the libraries. The other artifacts (docs etc.) are most easily built with Nix,
-so we recommend installing it regardless.
+The Haskell libraries in the Plutus Platform can be built in a number of ways.
+The prerequisites depend on how you want to build the libraries.
```

Comment:
Maybe we could rephrase slightly to be a bit more prescriptive for people that might need a hint; something like:

"The Haskell libraries can be built in a number of ways. Our recommendation is to use <<nix>> ..."